xl-v1-object.go

```diff
@@ -68,8 +68,12 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 	}
 
 	// If all the disks returned error, we return error.
-	if err := reduceErrs(errs); err != nil {
-		return toObjectErr(err, bucket, object)
+	if errCount, err := reduceErrs(errs); err != nil {
+		if errCount < xl.readQuorum {
```

Comment:
did you mean `if errCount > xl.readQuorum {` ?

already a `if !isDiskQuorum(errs, xl.readQuorum) {}` check happens just before this, so returning `err` here will give more insight on the error type than a generic errXLReadQuorum.
For example let's say, all the disks returned errVolumeNotFound, we should return errVolumeNotFound and not errXLReadQuorum